ThemeHelper.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * ThemeHelper Class Doc Comment.
+ *
+ * Include this in the theme file to access the helper methods.
+ *
+ * @category Class
+ * @package gent_base
+ * @author Bart Delrue
+ */
+class ThemeHelper {
+
+  /**
+   * @param string $key
+   *   The view mode key.
+   *
+   * @return mixed
+   *   Returns the view mode if found.
+   */
+  public static function isViewMode($key) {
```

Comment:
Why a static function?

Reply:
Because there is no reason for a class constructor.